zips/zip-0400.rst

```diff
@@ -43,46 +43,61 @@ Here ``zkey`` is the property name located at the first position of the database
 Schema
 ------
 
-According to Zcash v3.0.0-rc1 the following key-values can be found: the property names in **bold** mean only one instance of this type can exist in the entire database, while the others, suffixed by '\*' can have multiple instances. Keys and Values columns of the table contain the types that the stored data is representing. Included also there are the variable names hoping it will add some clarity to what the stored data is representing.
+The following key-values can be found: the property names in **bold** mean only one instance of this type can 
+exist in the entire database, while the others, suffixed by '\*' can have multiple instances. Keys and Values columns of the table
+contain the types that the stored data is representing. The Version column indicates the zcashd version(s) that create databases containing this key.
+Included also there are the variable names hoping it will add some clarity to what the stored data is representing.
 
 .. csv-table::
-   :header: "Name", "Description", "Keys", "Values"
+   :header: "Version", "Name", "Description", "Keys", "Values"
+   :align: left
 
-   "acc*", "Account data", "1. ``string strAccount``", "1. ``CAccount account``"
-   "acentry*", "Track internal transfers between accounts in the same wallet", "1. ``string strAccount``
+   "3..4", "acc*", "Account data. This entry is used in Bitcoin and not in Zcash.", "1. ``string strAccount``", "1. ``CAccount account``"
+   "3..4", "acentry*", "Track internal transfers between accounts in the same wallet. This entry is used in Bitcoin and not in Zcash.", "1. ``string strAccount``
    2. ``uint64_t nNumber``", "1. ``CAccountingEntry acentry``"
```

Comment:
Does this mean that these fields will never be present in a Zcash walletdb? In that case just delete them.
```suggestion
```